src/components/InventoryV2/view/InventorySeedlingsForNurseryTable.tsx

```diff
@@ -0,0 +1,410 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Typography, Grid, Box, useTheme, Theme, Popover } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import { Button, TableColumnType, Tooltip } from '@terraware/web-components';
+import { TopBarButton } from '@terraware/web-components/components/table';
+import strings from 'src/strings';
+import useDebounce from 'src/utils/useDebounce';
+import { FieldNodePayload, SearchNodePayload, SearchResponseElement, SearchSortOrder } from 'src/types/Search';
+import useDeviceInfo from 'src/utils/useDeviceInfo';
+import useForm from 'src/utils/useForm';
+import { InventoryFiltersType } from '../InventoryFiltersPopover';
+import { NurseryBatchService } from 'src/services';
+import useSnackbar from 'src/utils/useSnackbar';
+import { APP_PATHS } from 'src/constants';
+import { useLocalization, useOrganization } from 'src/providers';
+import Table from 'src/components/common/table';
+import { SortOrder } from 'src/components/common/table/sort';
+import FilterGroup, { FilterField } from 'src/components/common/FilterGroup';
+import { convertFilterGroupToMap, isBatchEmpty } from 'src/components/InventoryV2/FilterUtils';
+import Search from 'src/components/InventoryV2/Search';
+import DeleteBatchesModal from './DeleteBatchesModal';
+import BatchesCellRenderer from './BatchesCellRenderer';
+
+interface InventorySeedlingsForNurseryTableProps {
+  nurseryId: number;
+  modified: number;
+  setModified: (val: number) => void;
+  openBatchNumber: string | null;
+  onUpdateOpenBatch: (batchNum: string | null) => void;
+}
+
+const columns = (): TableColumnType[] => [
+  { key: 'batchNumber', name: strings.SEEDLING_BATCH, type: 'string' },
+  { key: 'species_commonName', name: strings.SPECIES, type: 'string' },
```

Comment:
Should we show the scientific name instead? Because common name is optional, scientific is always present

Reply:
That's a good question, I'll ask in Slack.